tests/xtro-sharpie/macOS-InputMethodKit.ignore

```diff
@@ -1,83 +1,14 @@
-## wip https://github.com/xamarin/xamarin-macios/pull/1930
-## blocked by lack of sample for validation
-
-!missing-field! IMKCandidatesOpacityAttributeName not bound
-!missing-field! IMKCandidatesSendServerKeyEventFirst not bound
+# introspections says that they return null strings.
 !missing-field! IMKControllerClass not bound
 !missing-field! IMKDelegateClass not bound
 !missing-field! IMKModeDictionary not bound
-!missing-field! kIMKCommandClientName not bound
-!missing-field! kIMKCommandMenuItemName not bound
-!missing-protocol! IMKMouseHandling not bound
-!missing-protocol! IMKStateSetting not bound
-!missing-selector! IMKCandidates::attachChild:toCandidate:type: not bound
-!missing-selector! IMKCandidates::candidateIdentifierAtLineNumber: not bound
-!missing-selector! IMKCandidates::candidateStringIdentifier: not bound
-!missing-selector! IMKCandidates::detachChild: not bound
-!missing-selector! IMKCandidates::initWithServer:panelType: not bound
-!missing-selector! IMKCandidates::initWithServer:panelType:styleType: not bound
-!missing-selector! IMKCandidates::lineNumberForCandidateWithIdentifier: not bound
-!missing-selector! IMKCandidates::selectCandidate: not bound
-!missing-selector! IMKCandidates::selectCandidateWithIdentifier: not bound
-!missing-selector! IMKCandidates::setAttributes: not bound
-!missing-selector! IMKCandidates::setCandidateData: not bound
-!missing-selector! IMKCandidates::setCandidateFrameTopLeft: not bound
-!missing-selector! IMKCandidates::setDismissesAutomatically: not bound
-!missing-selector! IMKCandidates::setPanelType: not bound
-!missing-selector! IMKCandidates::setSelectionKeys: not bound
-!missing-selector! IMKCandidates::setSelectionKeysKeylayout: not bound
-!missing-selector! IMKCandidates::show: not bound
-!missing-selector! IMKCandidates::showAnnotation: not bound
-!missing-selector! IMKCandidates::showSublist:subListDelegate: not bound
-!missing-selector! IMKInputController::annotationSelected:forCandidate: not bound
-!missing-selector! IMKInputController::candidateSelected: not bound
-!missing-selector! IMKInputController::candidateSelectionChanged: not bound
-!missing-selector! IMKInputController::compositionAttributesAtRange: not bound
-!missing-selector! IMKInputController::doCommandBySelector:commandDictionary: not bound
-!missing-selector! IMKInputController::initWithServer:delegate:client: not bound
-!missing-selector! IMKInputController::markForStyle:atRange: not bound
-!missing-selector! IMKInputController::setDelegate: not bound
-!missing-selector! IMKServer::initWithName:bundleIdentifier: not bound
-!missing-selector! IMKServer::initWithName:controllerClass:delegateClass: not bound
+
+# this are bound as IMKServerInput informal protocol
```

Comment:
When do we use informal protocol?

Reply:
An informal protocol is the same as a category; _informal protocol_ is the old name that was used before Apple decided to call them categories (and depending on how old Apple's documentation for a particular API is, the documentation will say either _informal protocol_ (old docs) or _category_ (newer docs)).

To answer your question: it depends on how it's declared in the header.